tools/cpp/CROSSTOOL

```diff
@@ -183,8 +183,11 @@ toolchain {
   compiler_flag: "external/org_chromium_clang_mac/lib/clang/6.0.0/include"
   compiler_flag: "-isystem"
   compiler_flag: "/usr/include"
+  compiler_flag: "-isystem"
```

Comment:
is this flag duplicated?

Reply:
I believe the `compiler_flag` group is collapsed into a statement so the above would become something like this:

```
clang ... -nostdinc++ \
  -isystem external/org_chromium_clang_mac/include/c++/v1 \
  -isystem external/org_chromium_clang_mac/lib/clang/6.0.0/include \
  -isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include
```

etc.